src/Workspaces/MSBuild/Test/NetCoreTests.cs

```diff
@@ -94,6 +96,27 @@ public async Task TestOpenProject_NetCoreApp()
         Assert.Empty(diagnostics);
     }
 
+    [ConditionalFact(typeof(DotNetSdkMSBuildInstalled))]
+    [Trait(Traits.Feature, Traits.Features.MSBuildWorkspace)]
+    [Trait(Traits.Feature, Traits.Features.NetCore)]
+    public async Task TestOpenInMemoryProject_NetCoreApp()
+    {
+        CreateFiles(GetNetCoreAppFiles());
+
+        var projectFilePath = GetSolutionFileName("Project.csproj");
+        var content = File.ReadAllText(projectFilePath);
+        var projectDir = Path.GetDirectoryName(projectFilePath);
+
+        await using var buildHostProcessManager = new BuildHostProcessManager(ImmutableDictionary<string, string>.Empty);
+
+        var buildHost = await buildHostProcessManager.GetBuildHostWithFallbackAsync(projectFilePath, CancellationToken.None);
```

Comment:
...like right here. :smile: But maybe this case is OK for your actual needs?

Reply:
It sounds like a good idea to just specify the NetCore host, here and in its practical usage in the LanguageServer.